net-misc/igmpproxy/files/igmpproxy.initd-r1

```diff
@@ -0,0 +1,19 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+config="/etc/${RC_SVCNAME}.conf"
+name="IGMPproxy"
+
+command="igmpproxy"
+command_args="${IGMPPROXY_OPTS} ${config}"
+procname="${RC_SVCNAME}"
```

Comment:
instead of using `procname` you can run `igmproxy` with `-n` flag and set `command_background="true"` and `pidfile`. openrc will create pidfile for you and properly track it.

Reply:
Well, is this a good idea to run a daemon on foreground and put it background with openrc?Anyway, updated the PR, to match it.

Reply:
It's totally fine. systemd does that all the time and many daemons in the tree handle it just fine. sometimes it's even better security-wise than letting the daemon create the pidfile. 
openrc creates pidfile as root and if daemon is configured to run as different user, the pidfile remains owned as root.